### PR TITLE
Fix the incorrect reference to the "Azure DDoS Protection Standard should be enabled" control closes #87

### LINKED ISSUE #87: Wrong Query for "Control: Azure DDoS Protection Standard should be enabled"
**Describe the bug**
The site refers to another query.

**To reproduce**
Visit https://hub.steampipe.io/mods/turbot/azure_compliance/controls/control.network_security_group_udp_service_restricted?context=benchmark.nist_sp_800_53_rev_5/benchmark.nist_sp_800_53_rev_5_sc/benchmark.nist_sp_800_53_rev_5_sc_5
the query is pointed to https://github.com/turbot/steampipe-mod-azure-compliance/blob/v0.8/regulatory_compliance/network.sp#L90-L98

**Expected behavior**
A query for DDOS Protection Standard should be used.

### PR DESCRIPTION
**Pull request by misraved**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: query/network/network_ddos_enabled.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+select
+  -- Required Columns
```
Comment (khushboo9024) [inline query/network/network_ddos_enabled.sql:2]: This query needs modification, control result is not matching with the policy result in azure portal.

Comment (khushboo9024) [reply query/network/network_ddos_enabled.sql:2]: May be we can use the below query:
```with application_gateway_subnet as (
  select
		distinct ( split_part(c -> 'properties' -> 'subnet' ->> 'id', '/', 9) ) as vn_name
	from
		azure_application_gateway as ag,
		jsonb_array_elements(gateway_ip_configurations) as c
)
select
  -- Required Columns
  a.name as resource,
  case
    when b.vn_name is null then 'ok'
    when b.vn_name is not null and enable_ddos_protection::bool then 'ok'
    else 'alarm'
  end as status,
  case
    when b.vn_name is null then 'DDoS protection not required.'
    when b.vn_name is not null and enable_ddos_protection::bool then a.name || ' DDoS protection enabled.'
    else a.name || ' DDoS protection disabled.'
  end as reason,
  -- Additional Dimensions
  resource_group,
  sub.display_name as subscription
from
  azure_virtual_network as a
  left join application_gateway_subnet as b on a.name = b.vn_name
  join azure_subscription sub on sub.subscription_id = a.subscription_id
  ```

File: query/network/network_ddos_enabled.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
+with application_gateway_subnet as (
+  select
+		distinct ( split_part(c -> 'properties' -> 'subnet' ->> 'id', '/', 9) ) as vn_name
```
Comment (rajlearner17) [inline query/network/network_ddos_enabled.sql:3]: Check formatting

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**